test/commands/validate.test.ts

```diff
@@ -31,7 +31,7 @@ describe('validate', () => {
       .command(['validate', './test/not-found.yml'])
       .it('should throw error if file path is wrong', (ctx, done) => {
         expect(ctx.stdout).to.equals('');
-        expect(ctx.stderr).to.equals('ValidationError: There is no file or context with name "./test/not-found.yml".\n');
+        expect(ctx.stderr).to.equals('Error Loading Specification From File: ./test/not-found.yml is an invalid file path\n');
```

Comment:
make sure we do not do always upper case in these error messages, it should be like regular sentence

```suggestion
        expect(ctx.stderr).to.equals('Error loading AsyncAPI document from file: ./test/not-found.yml is an invalid file path\n');
```